build.gradle.kts

```diff
@@ -4,6 +4,10 @@ plugins {
   kotlin("plugin.jpa") version "1.5.10"
 }
 
+configurations {
+  implementation { exclude(module = "spring-boot-graceful-shutdown") }
```

Comment:
not needed, spring boot comes supplied with own version these days that's already enabled